sw/device/tests/dif/README.md

```diff
@@ -0,0 +1,28 @@
+# The OpenTitan DIFs (Device Interface Functions) Library unit tests
```

Comment:
I think this file needs some examples of what these tests would look like in practice.